### DISCUSSION TITLE
May 28 Discussion

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/writers-workshop#10] [New class]
# the goal of the workshop

* notebooking and honesty

## teacher's preferred ground rules

* How to use the raise handle feature? Keeping the professor on track..
* I encourage y'all to interrupt me when I'm talking or demo'ing. 
  Sometimes I get distracted by my own special interests and this class if for you all.
* To help take notes, please respond to this discussion item and continue editing your one reply during class. i can see the changes on the discussion board as they happen.

## [call to action] :+1: :-1:  :smile: :confused:  :rocket:  :heart:  :eyes: :tada: attendance

react to this discussion so we can take attendance.

![image](https://user-images.githubusercontent.com/4236275/117461813-fd163180-af1b-11eb-901b-bfdee65d8bdc.png)

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

### Q2 how do you want to participate

* :heart:   i'm happy to be at the writers workshop this week
* :eyes: i want to just watch and listen
* :tada: i am okay being called on for discussions

## agenda

0. [:00-:07] chit chat
  ## assignment

4. [:50-:60] wrapup

    edit and improve your story in the third person, separate yourself from your work.

**Comment by isabela-pf** [comment]
Figuring out next goals:
- Will assignments work?
- Do we set group goals? Individual goals?
- Where does the writing go in the end? quansight.github.io? The main quansight blog.

Assignment:
Write a document that uses line and/or cell magics.
or
Write your own magics.

[Cool polyglot notebook](https://nbviewer.jupyter.org/gist/fperez/5b49246af4e340c37549265a90894ce6) by Fernando Pérez 

[Magics example](https://gist.github.com/tonyfast/3440a5b4c19a257d6164fff93fbef154) by Tony
